### PR TITLE
Custom Encounter Search

### PR DESCRIPTION
**Pull request by Nandita92**
Description
----

PR Checklist
----

- [x] Screenshot(s) of changes attached before changes merged.

Screenshot 2023-09-14 at 1 15 16 PM
393" alt="Screenshot 202
Screenshot 2023-09-14 at 1 15 51 PM
3-09-14 at 1 15
Screenshot 2023-09-14 at 1 15 00 PM

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/financial/general/account.md [hunk 1]
```diff
@@ -108,6 +113,24 @@ Notes:
...
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Account?_id=G71248138
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_account_search_guarantor_id) %>
+
+#### Request - `-encounter`
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Account??encounter=98028029&type=guarantor-balance
```
Comment (SantanuBanerjee) [inline content/millennium/r4/financial/general/account.md:127]: There are 2 question marks after /Account

**Review by SantanuBanerjee — APPROVED** [review]

**Review by BhavyaAcharya — APPROVED** [review]

File: content/millennium/r4/financial/general/account.md [hunk 2]
```diff
@@ -82,16 +84,19 @@ _Implementation Notes_
  `type`       | See notes | [`token`]     | The specific type of account. Example: `financial-account`
  `patient`    | See notes | [`reference`] | The entity that caused the expenses. Example: `Patient/12345`
  `-guarantor` | See notes | [`reference`] | The parties responsible for balancing the account. Example: `6330015-6330017`
+`-encounter`  | See notes | [`reference`] | The id to identify the encounter. Example: `98028029`
```
Comment (ashu-bdvt) [inline content/millennium/r4/financial/general/account.md:87]: Needs 1 more space to align with the above lines

File: content/millennium/r4/financial/general/account.md (hunk 1 above)
Comment (ashu-bdvt) [reply content/millennium/r4/financial/general/account.md:127]: +1

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Also update the PR title and do not include Jira IDs per the best practices defined. Since this is public repo we should not include the Jira #

File: content/millennium/r4/financial/general/account.md (hunk 1 above)
Comment (Nandita92) [reply content/millennium/r4/financial/general/account.md]: Updated here: https://github.com/cerner/fhir.cerner.com/commit/ff9aada138eefe2277802799a9124a96e5611328#diff-c95b06a9fe9f63e4cd5352c5f6cb27d71a28bcef3c8d6226c833a912332f4556R127

**Pull request closed without merging**